client/my-sites/site-settings/settings-reading/main.tsx

```diff
@@ -157,35 +154,20 @@ const ReadingSettingsForm = wrapSettingsForm( getFormSettings )(
 						isSavingSettings={ isSavingSettings }
 						updateFields={ updateFields }
 					/>
-					{ config.isEnabled( 'settings/newsletter-settings-page' ) ? (
```

Comment:
Is it expected to remove the feature flag condition as part of this PR?

Reply:
I think this is OK. The new checking is either it is `isSimpleSite` or blog sticker is enabled (to test Atomic & Jetpack).

Reply:
Yes, the flag is in production already. Instead of me fixing up types on the NewsletterSettingsSection component that is redundant here now, it made more sense to simply get rid of it. 🙂